docs/credits.md

```diff
@@ -504,7 +504,7 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | find-up                                           | Sindre Sorhus                                                             | [https://github.com/sindresorhus/find-up](https://github.com/sindresorhus/find-up)                                                                                                                                          | [MIT](https://opensource.org/licenses/MIT)                                                                          |
 | findup-sync                                       | Gulp Team                                                                 | [https://github.com/gulpjs/findup-sync](https://github.com/gulpjs/findup-sync)                                                                                                                                              | [MIT](https://opensource.org/licenses/MIT)                                                                          |
 | flow-parser                                       | Flow Team                                                                 | [https://github.com/facebook/flow](https://github.com/facebook/flow)                                                                                                                                                        | [MIT](https://opensource.org/licenses/MIT)                                                                          |
-| FluentValidation                                  | Jeremy Skinner                                                            | [http://github.com/JeremySkinner/fluentvalidation](http://github.com/JeremySkinner/fluentvalidation)                                                                                                                        | [Apache 2.0](https://github.com/JeremySkinner/FluentValidation/blob/master/License.txt)                             |
+| FluentValidation                                  | Jeremy Skinner                                                            | [https://fluentvalidation.net/](https://fluentvalidation.net/)                                                                                                                                                              | [Custom](https://www.nuget.org/packages/FluentValidation/8.5.1/license)                                             |
```

Comment:
Bah, that's a bit rubbish. Need to figure out how to make it realise from that link that its an apache 2.0 license